tests_requirements.txt

```diff
@@ -4,7 +4,9 @@ elasticsearch_dsl==7.4.0
 jinja2==3.0.3
 mock
 moto==2.2.16
+openshift-client==1.0.14
 pandas
+prometheus-api-client==0.5.0
```

Comment:
Please add it to Setup.py instead of  tests_requirements.txt
tests_requirements.txt is only for tests

Reply:
Done.